PythonTool/TagExpansion.py

```diff
@@ -29,13 +31,17 @@ def parse_arguments():
                         type=str,
                         help="Provide a name for the main directory."
                         )
+    parser.add_argument("-e", "--element",
+                        type=str,
+                        help="Provide a type of MB element other than hex."
+                        )
 
     args = parser.parse_args()
 
     return args
 
 
-def get_tag_lists(mb, element):
+def get_tag_lists(mb, element_type, element_id):
```

Comment:
`element_id` here meaning the user can supply the `GLOBAL_ID` of an element to use as the representative element?